tests/test_pip_step_decorator.py

```diff
@@ -0,0 +1,42 @@
+"""Test pip step decorator."""
```

Comment:
maybe worth combining this with the `test_environment.py` to iterate over flow names? Saves maintaining parallel identical tests

Reply:
I was trying to keep features a bit separated, but there's always some overlap.

In this case it's whether to test `pip_flow.py` for "project" environment in this file or in `test_environment.py`.

But testing `pip_flow.py` with `METAFLOW_DEFAULT_ENVIRONMENT=local` doesn't belong in `test_environment.py`.